projects/angular/src/utils/enums/key-codes.enum.ts

```diff
@@ -26,8 +26,4 @@ export enum IEKeyCodes {
   ArrowLeft = 'Left',
   Space = 'Spacebar',
   Escape = 'Esc',
-  Home = 'Home',
-  End = 'End',
-  Enter = 'Enter',
-  Tab = 'Tab',
```

Comment:
As this is a utility, I'd prefer to keep the unused enumeration values for completeness.

Reply:
These aren't different for IE, so they're not needed. See the other enum.